### DISCUSSION TITLE
Suggestion: add catalan language translation possibility

### DISCUSSION THREAD

**Discussion by woonaval** [voxeltycoon/issues#468] [UI&UX]
Hello,

I would like to translate the game to my main language - Catalan, but seems like it's not listed in Weblate. Would it  be possible to add it there?

Thank you

**Comment by victorleblais** [comment]
Translations are made by the community, anyone can make suggestions on Discord, but only can only be saved by someone that has the rights to do it, usually an active community member that asks to help. If you want to do that, ping @refreshfr on Discord in the #translation channel
